### PR TITLE
Fix a bug in #23.

### PR DESCRIPTION
**Pull request by SaltyChiang**
Fix the issue referred here: https://github.com/Endilll/vapoursynth-preview/pull/23#issuecomment-826439141

### CODE REVIEW

**Comment by Ichunjo** [comment]
It seems to work but when reloading, the checkbox "Sync Outputs" is always disabling.

**Comment by SaltyChiang** [comment]
> It seems to work but when reloading, the checkbox "Sync Outputs" is always disabled itself.

Fixed.
Thanks for pointing out the problem.

File: vspreview/core/abstracts.py [hunk 1]
```diff
@@ -142,8 +142,9 @@ def __setstate__(self, state: Mapping[str, Any]) -> None:
                 'Storage loading: Toolbar: failed to parse toggle')
             toggle = self.main.TOGGLE_TOOLBAR
 
-        if toggle:
-            self.toggle_button.click()
+        if not self.toggle_button.isChecked():
+            if toggle:
+                self.toggle_button.click()
```
Comment (Endilll) [inline vspreview/core/abstracts.py:147]: why not simply `self.toggle_button.setChecked(toggle)`?

File: vspreview/main.py [hunk 2]
```diff
@@ -328,6 +329,7 @@ class MainWindow(AbstractMainWindow):
     SEEK_STEP                 =     1  # frames
     STATUSBAR_MESSAGE_TIMEOUT =     3 * 1000  # s
     STORAGE_BACKUPS_COUNT     =     2
+    SYNC_OUTPUTS              = False
```
Comment (Endilll) [inline vspreview/main.py:332]: this constant should also be used when sync_output checkbox is created

**Review by Endilll — CHANGES_REQUESTED** [review]

File: vspreview/core/abstracts.py (hunk 1 above)
Comment (SaltyChiang) [reply vspreview/core/abstracts.py:147]: Just set the button checked will not toggle the toolbar.
In fact, this operation will just make the button pressed without any other effect.

File: vspreview/main.py (hunk 2 above)
Comment (SaltyChiang) [reply vspreview/main.py:333]: Done.

File: vspreview/core/abstracts.py (hunk 1 above)
Comment (Endilll) [reply vspreview/core/abstracts.py:147]: what about `if self.toggle_button.isChecked() != toggle:` then? to make it work both ways

Comment (SaltyChiang) [reply vspreview/core/abstracts.py:147]: Well, that's a bit subtle.
Imagine that for a toolbar, `toggle == False` (in the YAML file) but the `toggle_button` is checked, with autosave checkbox unchecked:
when you reload script, the parameter in the YAML file will not change and this toolbar will be set to invisible. 
Changing the toolbar layout when reloading shouldn't be a good idea for users.

Comment (SaltyChiang) [reply vspreview/core/abstracts.py:147]: There exists an assumption: the toolbars are always invisible when they are created. And the code ensures this.
Maybe `if not self.toggle_button.isChecked() and toggle:` is acceptable.

**Review by Endilll — APPROVED** [review]

**Pull request merged**